airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -60,8 +60,13 @@ properties:
     description: the Airbyte Protocol version supported by the connector
   normalizationRepository:
     type: string
+    description: an optional field indicating the name of the repository to be used for normalization. If the value of the flag is NULL - normalization is not used.
   normalizationTag:
     type: string
+    description: an optional field indicating the tag of the repository to be used for normalization.
   supportsDbt:
     type: boolean
-    default: false
+    description: an optional flag indicating whether DBT is used in the normalization. If the flag value is NULL - DBT is not used.
+  normalizationIntegrationType:
```

Comment:
The current format feels a bit error prone, normalization is defined by (`normalizationRepository`, `normalizationTag` and `normalizationIntegrationType`). If any of those three values is missing, normalization is disabled silently.
Does it make sense to extract this into an optional object that would have 3 required properties to reinforce this all or nothing effect?

Reply:
It's an interesting idea, but I suggest leaving it as is. Moreover, I added the necessary checks to prevent NPE